integreat_cms/cms/models/contact/contact.py

```diff
@@ -61,7 +63,37 @@ def __str__(self) -> str:
 
         :return: A readable string representation of the contact
         """
-        return f"{self.title} {self.name}"
+        location_name = str(self.location)
+        additional_attribute = self.get_additional_attribute()
+        return " ".join(part for part in [location_name, additional_attribute] if part)
+
+    def get_additional_attribute(self) -> str:
+        """
+        This function determines which string is shown for the contact
+        """
+        string = None
+
+        if self.point_of_contact_for:
+            string = _("with point of contact for: {}").format(
+                self.point_of_contact_for
+            )
+
+        if self.name:
+            string = _("with name: {}").format(self.name)
+
+        if self.email:
+            string = _("with email: {}").format(self.email)
+
+        if self.phone_number:
+            string = _("with phone number: {}").format(self.phone_number)
+
+        if self.website:
+            string = _("with website: {}").format(self.website)
+
+        if string:
+            return f" {string}"
+
+        return ""
```

Comment:
I'm not sure whether I understood the agreement with @osmars correctly, but we want to show "Contact for test-location with **point of contact: Integration specialist**" rather than "Contact for test-location with **website: https://integreat-app.de/**" if both point of contact and website are given, right? Currently the later is shown.

Reply:
Very true